onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -380,6 +556,8 @@ def quantize_model(self):
                         self.tensor_to_its_receiving_nodes[tensor_name] = []
                     self.tensor_to_its_receiving_nodes[tensor_name].append(node)
 
+        self.initializer_quant_params = self._calc_initializer_quant_params()
+        self._adjust_weight_quant_params_for_bias_tensors()
         self._quantize_normal_tensors()
         self._quantize_sharing_param_tensors()
         if self.quantize_bias:
```

Comment:
**Note**: This is the central model quantization loop. Previously, we computed scale/zp _and_ quantized initializers at the same time within `self._quantize_normal_tensors()`. Now, we separate the computation of the initializer's scale/zp from the actual quantization. This allows us to modify the quantization parameters (e.g., to adjust them for a bias) separately.

Additionally, as the code grew, the logic for determining the quantization type/symmetry/range for initializers was scattered throughout various functions and files. Now, all of the logic for determining the scale/zp of initializers is consolidated into a single function: `self._calc_initializer_quant_params()`.